src/katex.less

```diff
@@ -207,6 +207,34 @@
         min-width: 2px;
     }
 
+    .vbox {
+        display: -ms-inline-flexbox;
+        display: inline-flex;
+        -ms-flex-direction: column;
+        flex-direction: column;
+        align-items: baseline;
+    }
+
+    .hbox {
+        display: -ms-inline-flexbox;
+        display: inline-flex;
+        -ms-flex-direction: row;
+        flex-direction: row;
+        width: 100%;
+    }
+
+    .thinbox {
+        display: inline-flex;
+        flex-direction: row;
+        width: 0;
+        max-width: 0; // necessary for Safari
+    }
+
+    .glue {
+        flex-grow: 1;
+        flex-shrink: 1;
+    }
```

Comment:
This doesn't seem to be used anywhere.

Reply:
Correct. `.glue` is not used here. It would be necessary for use in `makeVList()`. I was looking forward a little bit here. Do you want me to take it out?

Reply:
Yeah, let's take it out for now.  It'll be easy enough to add back later once we need it.